tests/functionaltests/Tests/NSURLConnection.mm

```diff
@@ -281,16 +281,14 @@ static void _testRequestWithURL(NSURLConnectionTestHelper* connectionTestHelper)
     LOG_INFO("Received HTTP response headers: %d", [httpResponse allHeaderFields]);
 
     // Make sure we received data.
-    ASSERT_EQ_MSG(NSURLConnectionDelegateDidReceiveData,
-                  [(NSNumber*)[connectionTestHelper.delegateCallOrder objectAtIndex:1] integerValue],
-                  "FAILED: didReceiveData should be the 2nd delegate to be called!");
+    ASSERT_TRUE_MSG([@(NSURLConnectionDelegateDidReceiveData) isEqual:[connectionTestHelper.delegateCallOrder objectAtIndex:1]],
```

Comment:
1>C:\WinObjC\tests\functionaltests\Tests\NSURLConnection.mm(313,5): error : 'testing' is not a class, namespace, or enumeration